NuimoSDK/NuimoBluetoothController.cs

```diff
@@ -127,7 +127,7 @@ private void BluetoothGattCallback(GattCharacteristic sender, GattValueChangedEv
         private bool ReadFirmwareVersion()
         {
             return ReadCharacteristicValue(CharacteristicsGuids.FirmwareVersionGuid, bytes =>
-                DispatchOnMainAsync(() => FirmwareVersionRead?.Invoke(Encoding.ASCII.GetString(bytes)))
+                DispatchOnMainAsync(() => FirmwareVersionRead?.Invoke(Encoding.GetEncoding("ASCII").GetString(bytes, 0, bytes.Length)))
```

Comment:
Yes. I mentioned this particular example here: https://github.com/getsenic/nuimo-windows/pull/8#issuecomment-237915047

It's a rather long post, so I'll quote the relevant part here:

> In fact, I had to make a minor change to NuimoBluetoothController.cs since Encoding.ASCII was not available in all the targeted frameworks. Also, the only overload of GetString that was available across all the selected targets, was the one where you provide index and length of the bytes you want to read.